Gemfile

```diff
@@ -12,9 +12,9 @@ gem 'puma',                '>= 4.3.5'
 gem 'rack-cors',           '>= 1.0.4'
 gem 'rails',               '~> 5.2.2'
 
-gem 'manageiq-messaging'
+gem 'manageiq-messaging', '~> 1.0.0'
 
-gem 'topological_inventory-core', '~> 1.1.5'
+gem 'topological_inventory-core', '~> 1.2.0'
```

Comment:
@lindgrenj6 I think there is no reason why ingress-api depends on core, can you try to remove it?

Reply:
I pulled it out and all the tests pass - and running a collector I'm not seeing any crashes :+1: nice catch